### PR TITLE
blog(embertimes#153): add writeup on asset urls in corber

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2020-06-19-the-ember-times-issue-153.md [hunk 1]
```diff
@@ -128,14 +129,15 @@ In addition to responsive design, container queries can also help you learn a fe
...
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+If you ever wanted to write native mobile applications using [Corber](http://corber.io/),
+you might have had questions on how url paths for your **assets** should look like. Especially if they are supposed to work both **natively and in the browser**.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+But fear not! You can now learn all about it in [Beto Cantu's (@betocantu93)](https://github.com/betocantu93) recent blog post ["Understanding Ember.js Public Assets Path for Browser and Cordova"](https://dev.to/betocantu93/understanding-ember-js-public-assets-path-jj8). The post not only explains how asset urls are managed in different version of Ember applications,
+but also shows you how to write your own handy helper to make url handling a breeze in your next Corber project.
+
+Read the full post [on Beto Cantu's blog](https://dev.to/betocantu93/understanding-ember-js-public-assets-path-jj8)!
```
Comment (ijlee2) [inline source/2020-06-19-the-ember-times-issue-153.md]:
```suggestion
But fear not! You can now learn all about it in [Beto Cantú's (@betocantu93)](https://github.com/betocantu93) recent blog post ["Understanding Ember.js Public Assets Path for Browser and Cordova"](https://dev.to/betocantu93/understanding-ember-js-public-assets-path-jj8). The post not only explains how asset urls are managed in different version of Ember applications,
but also shows you how to write your own handy helper to make url handling a breeze in your next Corber project.

Read the full post [on Beto Cantú's blog](https://dev.to/betocantu93/understanding-ember-js-public-assets-path-jj8)!
```

**Review by ijlee2 — APPROVED** [review]
Looks great, thank you for the write-up! I think the u in Cantú is spelled with an accent mark, so I suggested changes above.

File: source/2020-06-19-the-ember-times-issue-153.md (hunk 1 above)
Comment (jayjayjpg) [reply source/2020-06-19-the-ember-times-issue-153.md]: Thank you for the thoughtful update - I missed that earlier!

**Comment by jayjayjpg** [comment]
:bowing_woman:

**Pull request merged**